### PR TITLE
Surround searchbox inputs with a div for styling

### PR DESCRIPTION
**Pull request by maccabeelevine**
This is essentially a two-line PR that encloses the searchbox inputs with a named div so that they can be styled.  For example the line of inputs could be a flexbox or grid layout.

image

Currently those elements share the same parent element `<form id="searchForm"` with the search tabs (among other stuff), so they are difficult to style.

### CODE REVIEW

File: themes/bootstrap3/templates/search/searchbox.phtml
Comment (maccabeelevine) [inline themes/bootstrap3/templates/search/searchbox.phtml:101]: These styles are just an example and would be removed before merge.  The effect is to let the main search box grow on mobile widths to take up whatever space it has available, but to still fit the Find button on the same line if there is nothing else in between.  It's nice space saver.

File: themes/bootstrap3/templates/search/searchbox.phtml
Comment (maccabeelevine) [inline themes/bootstrap3/templates/search/searchbox.phtml]: These styles are just an example and would be removed before merge. The effect is to let the main search box grow on mobile widths to take up whatever space it has available, but to still fit the Find button on the same line if there is nothing else in between. It's nice space saver.

File: themes/bootstrap3/templates/search/searchbox.phtml
Comment (maccabeelevine) [inline themes/bootstrap3/templates/search/searchbox.phtml:214]: I'm not actually sure how much should be included within the div.  I think everything within it here is inline, and that the filters below are supposed to be below.

**Comment by demiankatz** [comment]
Thanks, @maccabeelevine -- I'll defer to @crhallberg on this one (though would also welcome opinions from others). Chris is out of office today and might need a little time to catch up after he returns tomorrow, but hopefully you'll hear from him fairly soon. :-)

**Review by crhallberg — APPROVED** [review]
This looks good! Don't forget to clear out the example styles!

**Comment by maccabeelevine** [comment]
> This looks good! Don't forget to clear out the example styles!

Thanks.  I've removed the example CSS.

**Comment by sturkel89** [comment]
I reviewed the test branch in all three themes by adding filters and resizing the browser window from widest to narrowest. I compared behavior between test and dev in each of those environments.

Everything looked identical to me between test and dev in each of the themes. Unless I'm missing something, I would say that this is ready to merge.

**Comment by demiankatz** [comment]
Thanks, everyone!

**Pull request merged**
